pkgs/development/libraries/libinih/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, fetchFromGitHub, meson, ninja }:
+
+stdenv.mkDerivation rec {
+  pname = "libinih";
+  version = "52";
+
+  src = fetchFromGitHub {
+    owner = "benhoyt";
+    repo = "inih";
+    rev = "r${version}";
+    sha256 = "0lsvm34zabvi1xlximybzvgc58zb90mm3b9babwxlqs05jy871m4";
+  };
+
+  mesonFlags = [
+    "-Ddefault_library=shared"
+    "-Ddistro_install=true"
+  ];
+
+  nativeBuildInputs = [
+    meson
+    ninja
+  ];
+
+  meta = with stdenv.lib; {
+    homepage = "";
```

Comment:
Why did you remove the homepage and changelog?

Reply:
As explained above, I didn't remove them, I just did not add them.